fasthttp/sentryfasthttp.go

```diff
@@ -66,10 +68,33 @@ func (h *Handler) Handle(handler fasthttp.RequestHandler) fasthttp.RequestHandle
 			client.SetSDKIdentifier(sdkIdentifier)
 		}
 
+		convertedHTTPRequest := convert(ctx)
+
+		options := []sentry.SpanOption{
+			sentry.WithOpName("http.server"),
+			sentry.ContinueFromRequest(convertedHTTPRequest),
+			sentry.WithTransactionSource(sentry.SourceRoute),
+		}
+
+		method := string(ctx.Method())
+
+		transaction := sentry.StartTransaction(
+			sentry.SetHubOnContext(ctx, hub),
```

Comment:
Maybe I'm missing something but this line doesn't seem to match the comment above:

```
		// Unlike for other integrations, we don't support getting an existing
		// hub from the current request context because fasthttp doesn't use the
		// standard net/http.Request and because fasthttp.RequestCtx implements
		// context.Context but requires string keys.
		hub := sentry.CurrentHub().Clone()
```

Reply:
Yeah we shall remove this